auth-docker-postgres-jwt/handler/auth.go

```diff
@@ -82,10 +82,12 @@ func Login(c *fiber.Ctx) error {
 		userModel, err = getUserByUsername(identity)
 	}
 
+	const dummyHash = "$2a$10$7zFqzDbD3RrlkMTczbXG9OWZ0FLOXjIxXzSZ.QZxkVXjXcx7QZQiC" // => Hashed " "
+
 	if err != nil {
 		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"status": "error", "message": "Internal Server Error", "data": err})
 	} else if userModel == nil {
-		CheckPasswordHash(pass, "")
+		CheckPasswordHash(pass, dummyHash)
```

Comment:
```suggestion
{
 		// Always perform a hash check, even if the user doesn't exist, to prevent timing attacks
 		 CheckPasswordHash(pass, dummyHash)
```